images/cockpit-images.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: v1
+kind: List
+items:
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: images
+    labels:
+      infra: cockpit-images
+  spec:
+    containers:
+      - name: images
+        image: cockpit/images
+        ports:
+          - containerPort: 8080
+            protocol: TCP
+        ports:
+          - containerPort: 8443
+            protocol: TCP
+        volumeMounts:
+        - name: secrets
+          mountPath: /secrets
+          readOnly: true
+        - name: images
+          mountPath: /cache/images
+        - name: httpd-log
+          mountPath: /var/log/nginx
+        - name: httpd-state
+          mountPath: /var/lib/nginx
+        - name: httpd-state-tmp
+          mountPath: /var/lib/nginx/tmp
```

Comment:
Can we have just one mount for the entire ```/var/lib/nginx``` directory? If there's a good reason to have separate that's fine ... but just thought I'd ask.

Reply:
That would be nicer indeed, but unfortunately does not work. Just creating /var/lib/nginx/ causes nginx to fail with a permission error to mkdir the /tmp dir below it, as the parent dir (/var/lib/nginx) is not writable to the random uid. For the same reason just creating /var/lib/nginx//tmp also fails because the parent dir is not accessible to the random uid.

Reply:
Makes sense.